system/include/emscripten/threading_legacy.h

```diff
@@ -49,17 +54,22 @@ typedef struct em_queued_call em_queued_call;
 
 // - starting from the lowest bits upwards, each pair of two subsequent bits
```

Comment:
```suggestion
// - starting from the lowest bits upwards, each triple of two subsequent bits
```

Reply:
I just adjusted this comment based on the changes in comments below it. Where is the modified code for this?

Reply:
Comment update was from a previous experiment.  Reverted this now.